src/dataMap.js

```diff
@@ -573,15 +572,15 @@ class DataMap {
   /**
    * Add/remove row(s) to/from the data source.
    *
-   * @param {number} index Physical index of the element to remove.
+   * @param {number} index Physical index of the element to add/remove.
    * @param {number} amount Number of rows to add/remove.
-   * @param {object} element Row to add.
+   * @param {...object} elements Row elements to be added.
    */
-  spliceData(index, amount, element) {
-    const continueSplicing = this.instance.runHooks('beforeDataSplice', index, amount, element);
+  spliceData(index, amount, ...elements) {
+    const continueSplicing = this.instance.runHooks('beforeDataSplice', index, amount, elements);
```

Comment:
You're right, it is. I did not think much of it, because it's currently used only in NestedRows, and wasn't even mentioned in the docs. But yeah, we should mention it in the docs of 8.0.0 (@scarletfog?)

Reply:
As it should be kept hidden there is nothing to describe. But thank you @wszymanski for being watchful! 💪